patches/third_party-blink-renderer-core-frame-navigator.h.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/third_party/blink/renderer/core/frame/navigator.h b/third_party/blink/renderer/core/frame/navigator.h
+index 4572df017cf40ca2f2b8b746ca1aeea3021a277c..f0a92768ffa13b9b4c7a9ec209ab9640cef26a7a 100644
+--- a/third_party/blink/renderer/core/frame/navigator.h
++++ b/third_party/blink/renderer/core/frame/navigator.h
+@@ -33,6 +33,7 @@
+ #include "third_party/blink/renderer/platform/heap/handle.h"
+ #include "third_party/blink/renderer/platform/supplementable.h"
+ #include "third_party/blink/renderer/platform/wtf/forward.h"
++#include "brave/third_party/blink/renderer/core/frame/navigator_brave.h"
+ 
+ namespace blink {
+ 
+@@ -71,6 +72,7 @@ class CORE_EXPORT Navigator final : public ScriptWrappable,
+ 
+   void Trace(blink::Visitor*) override;
+ 
++  BRAVE_NAVIGATOR_OVERRIDES
```

Comment:
this should be `BRAVE_NAVIGATOR_H` per https://github.com/brave/brave-browser/wiki/Patching-Chromium#subclass-and-override but I'm not convinced we need this patch yet